VMT/KCSA/KCSA-CVE-2020-2025.md

```diff
@@ -0,0 +1,47 @@
+announcement-date: 2020-05-28
+
+id: KCSA-CVE-2020-2025
+
+title: Kata Containers with Cloud Hypervisor guest image persists vulnerability
+
+description: Kata Containers before 1.11.0 on Cloud Hypervisor persists guest filesystem changes to the underlying image file on the host. A malicious guest can overwrite the image file to gain control of all subsequent guest VMs. Since Kata Containers uses the same VM image file with all VMMs, this issue may also affect QEMU and Firecracker based guests.
+
+affected-components:
+
+  - components: `kata-runtime`
+    version: Before v1.11.0
+
+vulnerabilities:
+
+  - cve-id: CVE-2020-2025
+
+reporters:
+
+  - name: Yuval Avrahami
+    affiliation: Yuval Avrahami
+    reported:
+      - CVE-2020-2025
+
+issues:
+
+  links:
+    - https://github.com/kata-containers/runtime/issues/2488
```

Comment:
Is this URL correct?

Reply:
This probably needs to be https://github.com/kata-containers/runtime/pull/2487 instead @bergwolf

Reply:
The PR link is below in reviews section. This one is the issue link, although it doesn't talk much about the vulnerability.